public/doc/openapi-3-v1.0.0.json

```diff
@@ -2635,7 +2635,7 @@
             "content": {
               "application/json": {
                 "schema": {
-                  "$ref": "#/components/schemas/DataDrivenFormSchema"
+                  "$ref": "#/components/schemas/ServicePlan"
```

Comment:
Do you know why https://projects.engineering.redhat.com/browse/SSP-1049 was returning a `None` schema object when we were declaring a Schema type?  I would have expected a different error than None.

Reply:
It's because I had the types mismapped - the deserialization was failing since the client was expecting just a `DataDrivenForm` schema rather than the `ServicePlan` schema.

Reply:
Basically the problem is that the GET /base and /modified schemas always return a `ServicePlan` object, as seen here: 
`/service_plans/:id/base`
https://github.com/lindgrenj6/catalog-api/blob/9bdfaad95d9fc4a4bffb7e120250b18529fe6538/app/controllers/api/v1/service_plans_controller.rb#L27-L28
`/service_plans/:id/modified`
https://github.com/lindgrenj6/catalog-api/blob/9bdfaad95d9fc4a4bffb7e120250b18529fe6538/app/controllers/api/v1/service_plans_controller.rb#L32-L38

Since the client was expecting just a raw `DataDrivenForm` object the de-serialization was failing, thus the `None` object.